vault/request_handling.go

```diff
@@ -1726,9 +1742,8 @@ func (c *Core) handleLoginRequest(ctx context.Context, req *logical.Request) (re
 // LoginCreateToken creates a token as a result of a login request.
 // If MFA is enforced, mfa/validate endpoint calls this functions
 // after successful MFA validation to generate the token.
-func (c *Core) LoginCreateToken(ctx context.Context, ns *namespace.Namespace, reqPath, mountPoint string, resp *logical.Response, loginRequestData map[string]interface{}) (bool, *logical.Response, error) {
+func (c *Core) LoginCreateToken(ctx context.Context, ns *namespace.Namespace, resp *logical.Response, reqPath, mountPoint, role string) (bool, *logical.Response, error) {
```

Comment:
Is this reordering of arguments necessary?

Reply:
no its not :), I just grouped the strings together, but can switch back if its more confusing.

Reply:
I think @ncabatoff meant:

```suggestion
func (c *Core) LoginCreateToken(ctx context.Context, ns *namespace.Namespace, reqPath, mountPoint, role string, resp *logical.Response) (bool, *logical.Response, error) {
```

Reply:
It's best when making a PR you plan to backport to minimize unnecessary changes in my opinion.

Reply:
to minimize changes I changed it back the original order. Removed `loginRequestData` and replaced it with `role`